package.json

```diff
@@ -49,6 +49,7 @@
     "@types/rbush": "4.0.0",
     "earcut": "^3.0.0",
     "geotiff": "^2.1.3",
+    "ol": "^10.6.1",
```

Comment:
Please remove, also from package-lock.json. This pull request does not need to contain any changes to `package.json` and `package-lock.json`.